invenio_files_rest/config.py

```diff
@@ -68,6 +68,14 @@
    with PostgreSQL database.
 """
 
+FILES_REST_OBJECT_TAG_VALUE_MAX_LEN = 255
+"""Maximum length of the ObjectTag.value.uri field.
```

Comment:
```suggestion
"""Maximum length of the ObjectTag.value field.
```

minor: I know you store `uri`s there, but not everyone does ;)